crc-admin-helper.spec.in

```diff
@@ -32,6 +32,7 @@ License:        MIT
 ExclusiveArch:  x86_64 aarch64
 URL:            %{gourl}
 Source0:        %{gosource}
+ExcludeArch:    ppc64le s390x
```

Comment:
In my opinion it's better to use `ExclusiveArch` to list the arches we want to build on.

Reply:
looks like ExclusiveArch is not working internal brew taskID=62069957 (reference) and it still build for all arch.

Reply:
This is caused by `gometa`, which sets `ExclusiveArch` to all the architectures supported by golang.
This should work even though it's not very nice.
```
diff --git a/crc-admin-helper.spec.in b/crc-admin-helper.spec.in
index 87250d9..29954a6 100644
--- a/crc-admin-helper.spec.in
+++ b/crc-admin-helper.spec.in
@@ -3,6 +3,8 @@
 %global goname          crc-admin-helper
 Version:                0.5.3
 
+# %gometa sets ExclusiveArch to %{golang_arches}, and we only want to build on x86_64 and aarch64
+%global golang_arches x86_64 aarch64
 %gometa
 
 %global gobuilddir %{_builddir}/%{archivename}/_build
```

Reply:
On my fedora, `ExclusiveArch` is set to `i386 i486 i586 i686 pentium3 pentium4 athlon geode x86_64 armv3l armv4b armv4l armv4tl armv5tl armv5tel armv5tejl armv6l armv6hl armv7l armv7hl armv7hnl armv8l armv8hl armv8hnl armv8hcnl aarch64 ppc64le s390x riscv64` when using `gometa`, if we use `ExcludeArch`, in my opinion we would need to list all of these...